firstpythonscript.py

```diff
@@ -2,11 +2,26 @@
 import numpy as np
 import sympy as sp
 
-x ,t = sp.symbols('x t')
+t, y = sp.symbols("y t")
```

Comment:
This is needs to be switched to the following:
```suggestion
t, y = sp.symbols("t y")
```
This is why I was getting incorrect results. After making this change on my local copy, I now see the correct output:
```bash
% python3 firstpythonscript.py
Enter ODE (in terms of t & y): y' =  -y+t
write an initial point for t(to): 59
write an initial point for y(yo): 5
The variables in the function are: 54